std/conv.d

```diff
@@ -81,8 +81,16 @@ private auto convError(S, T)(S source, string fn = __FILE__, size_t ln = __LINE_
     if (source.empty)
         msg = "Unexpected end of input when converting from type " ~ S.stringof ~ " to type " ~ T.stringof;
     else
-        msg =  text("Unexpected '", source.front,
+    {
+        import std.range : ElementType;
+        ElementType!S el = source.front;
+
+        if (el == '\n')
+            msg = text("Unexpected '\\n' when converting from type " ~ S.stringof ~ " to type " ~ T.stringof);
+        else
+            msg =  text("Unexpected '", el,
```

Comment:
> Functions cannot be interpreted while being compiled."

Hmm you can change this by switching to `format("...", source.take(1))`, but then you get nice error messages due to `@safe` and `pure` violations.
This can be hacked with sth. ugly like:

```
        () @trusted {
        msg = (cast(string function() pure @safe)(() =>
            text("Unexpected ", format("%(%s%)", source.take(1)),
            " when converting from type " ~ S.stringof ~ " to type " ~ T.stringof)))();
```

But then you get other errors - presumably to the fact that `to` is used somewhere in `format`